src/Dropdown/Dropdown.tsx

```diff
@@ -52,13 +60,14 @@ export const Dropdown: FC<Props> = ({
         <Select
           id={id}
           disabled={disabled}
-          defaultValue={list.length === 1 ? String(list[0].value) : placeholder}
-          onChange={(e: React.FormEvent<HTMLSelectElement>) =>
-            onSelect(e.currentTarget.value)
-          }
+          onChange={(e: React.FormEvent<HTMLSelectElement>) => {
+            setValue(e.currentTarget.value);
+            onSelect(e.currentTarget.value);
+          }}
           required
+          value={value}
         >
-          {list.length > 1 && (
+          {list.length !== 1 && (
```

Comment:
I'm trying to think of the UX in case the *list*'s length is `0`

Reply:
In fact, I think we might want to just remove that conditional rendering here, as there's no harm having a *hidden* placeholder when the length is 1 and the single *unhidden* item is auto-selected.

Reply:
The list can be empty in the Custom Car MTA flow, where dropdowns like engine capacity will be empty before the manufacturer and model has been selected.